RushHour/RushHourMod.cs

```diff
@@ -22,13 +22,17 @@ public void OnSettingsUI(UIHelperBase helper)
                 new OptionsCheckbox() { readableName = "EXPERIMENTAL: Use improved commercial demand", value = false, uniqueName = "UseImprovedCommercial" }
             };
 
+            loadSettingsFromSaveFile();
```

Comment:
Oh, that makes so much sense! It was my pleasure, thanks for the explanation. :)